keylime/ca_impl_cfssl.py

```diff
@@ -124,6 +124,7 @@ def mk_cacert():
 
 
 def mk_signed_cert(cacert, ca_pk, name, serialnum):
+    del cacert, serialnum
```

Comment:
Could we default to None to workaround this lint warning? It appears like implementation mismatch between cfssl and openssl, I don't know if it's possible to get a parity.

Reply:
Assigning None to cacert either in a line in the code or as a default value in the signature of the function doesn't resolve the issue of the variable being unused. The `del` seems the only way to resolve this.

Reply:
I think pylint is overly strict on this, optional arguments is, optional ...

Reply:
an optional argument that is unused is ... unused

Reply:
Yeah, and it makes consistent interface more harder, I hope lint tools help us find real issues, instead of being driven exhausted by it. Anyway, we can revisit in any time.

Reply:
An example of this is the signal handler, the signature is defined by std, not us, but there do have cases that we simply ignore.

Reply:
I think pylint will help to avoid certain code-related issues and provide better surveillance over the code so that issues don't even make it into the repo.